server/mscalendar/daily_summary.go

```diff
@@ -92,6 +93,65 @@ func (m *mscalendar) SetDailySummaryEnabled(user *User, enable bool) (*store.Dai
 	return dsum, nil
 }
 
+func (m *mscalendar) processAllDailySummaryWithIndividualCredentials(now time.Time, userIndex store.UserIndex) error {
```

Comment:
We may be able to de-duplicate some code between `ProcessAllDailySummary` and `processAllDailySummaryWithIndividualCredentials`. It would also be good to get this code under a unit test

Also it seems this block is missing, that exists in `ProcessAllDailySummary`:

https://github.com/mattermost/mattermost-plugin-mscalendar/blob/429698a2ca4e3680fdf12cf502f2c3ae02af089b/server/mscalendar/daily_summary.go#L231-L232